src/forms/datalist/DataList.tsx

```diff
@@ -82,6 +83,7 @@ type DataListProps = {
     helperText?: string;
     onChange?: (evt: React.ChangeEvent<HTMLInputElement>) => void;
     onBlur?: (evt: React.FocusEvent<HTMLInputElement>) => void;
+    autoComplete?: "on" | "off";
```

Comment:
Can we move these to enum and export that too.

Reply:
Done